popgen/mcmcSelection.go

```diff
@@ -165,6 +157,20 @@ func MetropolisHastings(data AFS, muZero float64, sigmaZero float64, iterations
 	}
 }
 
+func BuildBinomCache(allN []int) [][]float64 {
```

Comment:
Didn't we already have this function somewhere?

Reply:
This code previously existed, but wasn't wrapped up in a helper function. I needed this same code in another context, so I'm just repackaging here